gdx/src/com/badlogic/gdx/graphics/g2d/BitmapFont.java

```diff
@@ -101,6 +104,18 @@ public BitmapFont (FileHandle fontFile, TextureRegion region) {
 	public BitmapFont (FileHandle fontFile, TextureRegion region, boolean flip) {
 		this(new BitmapFontData(fontFile, flip), region, true);
 	}
+	
+	/** Creates a BitmapFont with the glyphs relative to the specified region. If the region is null, the glyph textures are loaded
+	 * from the image file given in the font file. The {@link #dispose()} method will not dispose the region's texture in this
+	 * case!
+	 * @param region The texture region containing the glyphs. The glyphs must be relative to the lower left corner (ie, the region
+	 *           should not be flipped). If the region is null the glyph images are loaded from the image path in the font file.
+	 * @param flip If true, the glyphs will be flipped for use with a perspective where 0,0 is the upper left corner.
+	 * @param distanceField If true, font will be rendered with distance field shader
+	 * @param distanceFieldSmoothing Smoothing factor for distance field shader */
+	public BitmapFont (FileHandle fontFile, TextureRegion region, boolean flip, boolean distanceField, float distanceFieldSmoothing) {
+		this(new BitmapFontData(fontFile, flip), region, true);
+	}
```

Comment:
Constructor doesn't use the parameters. Also, BitmapFont has too many constructors already. There is no need to specify all settings in the constructor, setters can be used.